client/gulpfile.js

```diff
@@ -53,8 +58,19 @@ gulp.task('icons:copy', () => {
     .pipe(gulp.dest(sources.icons.distPath));
 });
 
+// TODO move paths to vars and synchronous stuff
+gulp.task('cleanCSS', function () {
+  return gulp.src('../dist/assets/css/')
+    .pipe(clean({force: true}));
+});
+
+gulp.task('cleanJS', function () {
+  return gulp.src('../dist/assets/js/')
```

Comment:
Isn't this a reference to [this](https://github.com/wellcometrust/wellcomecollection.org/pull/457/files#diff-64bf274ee0d4e9c521933759c5e52d8fL24)?